test/OpenTelemetry.Instrumentation.Http.Tests/HttpClientTests.Basic.cs

```diff
@@ -462,6 +466,89 @@ public async Task HttpClientInstrumentationContextPropagation()
             Assert.Equal("b1=v1", baggages.Single());
         }
 
+        [Fact]
+        public async Task HttpClientInstrumentationReportsExceptionEventForNetworkFailuresWithGetAsync()
+        {
+            var exportedItems = new List<Activity>();
+            bool exceptionThrown = false;
+
+            using var traceprovider = Sdk.CreateTracerProviderBuilder()
+                   .AddHttpClientInstrumentation(o => o.RecordException = true)
+                   .AddInMemoryExporter(exportedItems)
+                   .Build();
+
+            using var c = new HttpClient();
+            try
+            {
+                await c.GetAsync("https://www.invalidurl.com");
```

Comment:
This is risky. 
This test may fail if someone registers this domain.

If you're just trying to have the httpclient throw an exception, I think you can use an invalid url. (eg `GetAsync("https://")` ).

If you're trying to force a network error specifically, I think you could use a non-routable ip address (eg `GetAsync("https://10.0.0.0/invalidurl")`).

Reply:
This one times out https://10.0.0.0/invalidurl. I have updated it to use something random which we use in other tests.